src/core/meshcop/dtls.cpp

```diff
@@ -68,7 +68,8 @@ Dtls::Dtls(ThreadNetif &aNetif):
     mSendHandler(NULL),
     mContext(NULL),
     mClient(false),
-    mMessageSubType(0)
+    mMessageSubType(0),
+    mMessageDefaultSubType(0)
```

Comment:
Initialize `mMessageSubType` and `mMessageDefaultSubType` with `Message::kSubTypeNone`?